docs/whatsnew-5.2.rst

```diff
@@ -330,6 +330,13 @@ older `azure-servicebus` versions.
 
 .. _v520-news:
 
+Dropped support for Pymongo 3.12.1 and above
```

Comment:
this could be like this "pymong 3.12.1 is not compatible with celery 5.2 due to this factors......"